fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1639,7 +1639,7 @@ void buildPartitions(Database db, OlapTable table, List<Partition> partitions) t
             numReplicas += partition.getReplicaCount();
         }
 
-        if (partitions.size() >= 3 && numAliveBackends >= 3 && numReplicas >= numAliveBackends * 500) {
+        if (true /*partitions.size() >= 3 && numAliveBackends >= 3 && numReplicas >= numAliveBackends * 500*/) {
```

Comment:
should use a FE config or session variable here.

Reply:
Done